servers/store/src/test/java/org/projectnessie/server/store/TestStoreWorker.java

```diff
@@ -399,6 +412,164 @@ void testSerde(Map.Entry<ByteString, Content> entry) {
         .isEqualTo(entry.getKey());
   }
 
+  private static JsonNode loadJson(String resource) {
+    URL url = Thread.currentThread().getContextClassLoader().getResource(resource);
```

Comment:
Why `getContextClassLoader` and not `TestStoreWorker.class.getClassLoader()`? I suppose the latter is a more stable representation of the generated resources. It maps to the test class path, which maps to module dependencies,  which include the resources.